trade-finance-manager-ui/templates/utilisation-reports/_macros/recorded-payments-for-selected-fee-records-table.njk

```diff
@@ -0,0 +1,41 @@
+{% from "govuk/components/accordion/macro.njk" import govukAccordion %}
+{% macro render(params) %}
```

Comment:
Other files have an empty line between imports and the macro, so could we add that here too please?